src/cdk/drag-drop/drag-ref.ts

```diff
@@ -386,6 +379,24 @@ export class DragRef<T = any> {
     this._passiveTransform = {x: 0, y: 0};
   }
 
+  /**
+   * Sets a handle as disabled. While a handle is disabled, it'll capture and interrupt dragging.
+   * @param handle Handle element that should be disabled.
+   */
+  disableHandle(handle: HTMLElement) {
```

Comment:
Note: I'm not too happy with having to add this method and `enableHandle` below, since it means having [to do this above](https://github.com/angular/material2/pull/14654/files#diff-417c7bfe86312f7b67aa286e5a0993b2R217), however we need them in order to keep backwards compatibility. They allow us to handle the case where the user starts to drag from a disabled handle, in which case the dragging sequence should be interrupted. In the future we can reconsider whether supporting that case is worth it.